src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -26,6 +26,18 @@ public interface JsonApiMetricsConfig {
   @WithDefault("sort.type")
   String sortType();
 
+  @NotBlank
+  @WithDefault("json.shredding.serialization.total.documents")
+  String jsonShreddingSerializedTotalDocuments();
+
+  @NotBlank
+  @WithDefault("json.shredding.serialization.size")
+  String jsonShreddingSerializedSize();
```

Comment:
Is this also a counter?

Reply:
@amorton can you comment on what we were hoping to track with this? Does it make more sense as a counter or some other metric type?